packages/hardhat-core/src/internal/hardhat-network/provider/provider.ts

```diff
@@ -458,9 +458,10 @@ export class EdrProviderWrapper
             balance: BigInt(account.balance),
           };
         }),
-        initialBaseFeePerGas: BigInt(
-          config.initialBaseFeePerGas ?? 1_000_000_000
```

Comment:
There is a test that checks that the base fee from the remote block is used if nothing is passed in as config, so I had to remove this.